compiler/rustc_typeck/src/check/method/probe.rs

```diff
@@ -1060,19 +1071,21 @@ impl<'a, 'tcx> ProbeContext<'a, 'tcx> {
         )))
     }
 
+    #[instrument(skip(self))]
```

Comment:
like this :)

Reply:
I reverted the changes related to debug logs now :)